widelands-build19-appdata-vercmp.patch

```diff
@@ -0,0 +1,29 @@
+diff -Naur a/debian/widelands.appdata.xml b/debian/widelands.appdata.xml
+--- a/debian/widelands.appdata.xml	2018-11-19 19:11:36.419195717 +0100
++++ b/debian/widelands.appdata.xml	2018-11-19 19:11:44.653254782 +0100
+@@ -174,25 +174,6 @@
+   <update_contact>widelands-dev_at_lists.launchpad.net</update_contact>
+   <releases>
+     <release date="2016-11-11" version="Build 19"/>
+-    <release date="2014-02-22" version="Build 18"/>
+-    <release date="2012-04-23" version="Build 17"/>
```

Comment:
Why this patch is needed?

Reply:
@bilelmoussaoui 

> Why this patch is needed?

I've already explained this.
https://github.com/flathub/flathub/pull/394#issuecomment-440044737

See also:
https://github.com/flatpak/freedesktop-sdk-images/issues/128
https://github.com/hughsie/appstream-glib/issues/270
https://github.com/hughsie/appstream-glib/pull/272